master/internal/job/job.go

```diff
@@ -102,8 +102,20 @@ func (j *Manager) GetJobs(
 		return nil, err
 	}
 
+	jobCount := len(jobQ)
+
+	// Try to fetch Non-DAI jobs if supported by the Resource Manager.
+	// If the Resource Manager returned Non-DAI jobs, increase the job count.
+	// Otherwise, we do not show any Non-DAI jobs.
+	nonDaiJobs, err := j.rm.GetNonDaiJobs(j.system, sproto.GetNonDaiJobs{
+		ResourcePool: resourcePool,
+	})
+	if err == nil {
+		jobCount += len(nonDaiJobs)
+	}
```

Comment:
check the error properly, if it is non-nil, just return an error.

Reply:
Updated as suggested